frontend/src/js/controllers/ourteamCtrl.js

```diff
@@ -0,0 +1,45 @@
+// Invoking IIFE for our team
+
+(function() {
+
+    'use strict';
+
+    angular
+        .module('evalai')
+        .controller('ourTeamCtrl', ourTeamCtrl);
+
+    ourTeamCtrl.$inject = ['utilities', '$state', '$rootScope', '$mdDialog'];
+
+    function ourTeamCtrl(utilities, $state, $rootScope, $mdDialog) {
+        /* jshint validthis: true */
+        var vm = this;
+
+        var parameters = {};
+        parameters.url = 'web/team/';
+        parameters.method = 'GET';
+        parameters.callback = {
+            onSuccess: function(response) {
+                var status = response.status;
+                var details = response.data;
+                if (status == 200) {
+                    var core_team_array = [];
+                    var contributing_team_array = [];
+                    for (var i = 0; i < details.length; i++) {
+                        if (details[i].team_type === "Core Team") {
+                            vm.core_team_type = details[i].team_type;
```

Comment:
@RishabhJain2018 Follow camel case rule for variable names. eg., core_team_type rename it to coreTeamType